### PR TITLE
Fix lint errors in rice-datanode

### PR DESCRIPTION
**Pull request by LINKIWI**
* Added `rice-datanode` to list of source directories to consider
* Fixed lint errors in all files under `rice-datanode`

Depends on #26.

### CODE REVIEW

File: rice-datanode/source/ClientDatanodeProtocolImpl.cc [hunk 1]
```diff
@@ -1,86 +1,82 @@
...
     : port(port_arg), server(port) {
   InitServer();
   LOG(INFO) << "Created client datanode translator.";
 }
 
-std::string ClientDatanodeTranslator::getReplicaVisibleLength(std::string input) {
-  GetReplicaVisibleLengthRequestProto req;
+std::string ClientDatanodeTranslator::getReplicaVisibleLength(
+  std::string input) {
+  hadoop::hdfs::GetReplicaVisibleLengthRequestProto req;
```
Comment (ghost) [inline rice-datanode/source/ClientDatanodeProtocolImpl.cc]: same here

File: rice-datanode/include/ClientDatanodeProtocolImpl.h [hunk 2]
```diff
@@ -30,18 +32,18 @@ class ClientDatanodeTranslator {
...
+
  private:
   std::string Serialize(google::protobuf::Message &);
   void InitServer();
   void RegisterClientRPCHandlers();
   void Config();
   void logMessage(google::protobuf::Message &req, std::string req_name);
 
-  FsServerDefaultsProto server_defaults;
+  hadoop::hdfs::FsServerDefaultsProto server_defaults;
```
Comment (ghost) [inline rice-datanode/include/ClientDatanodeProtocolImpl.h]: this works, but what i've been doing is changing:
`using namespace hadoop::hdfs`
to
`using hadoop::hdfs::FsServerDefaultsProto `
(and copying that for each specific import)
which is i think what the lint error is suggesting. what do you think is better?

File: rice-datanode/source/ClientDatanodeProtocolImpl.cc [hunk 3]
```diff
@@ -1,86 +1,82 @@
...
-  DeleteBlockPoolResponseProto res;
-  // TODO delete the block pool. Response contains no fields
+  hadoop::hdfs::DeleteBlockPoolResponseProto res;
+  // TODO(anyone): delete the block pool. Response contains no fields
   return Serialize(res);
 }
 
 std::string ClientDatanodeTranslator::getBlockLocalPathInfo(std::string input) {
-  GetBlockLocalPathInfoRequestProto req;
+  hadoop::hdfs::GetBlockLocalPathInfoRequestProto req;
```
Comment (ghost) [inline rice-datanode/source/ClientDatanodeProtocolImpl.cc]: etc

File: rice-datanode/include/ClientDatanodeProtocolImpl.h (hunk 2 above)
Comment (LINKIWI) [reply rice-datanode/include/ClientDatanodeProtocolImpl.h]: I have no preference.

I'll let you pick one and we'll standardize on that for the rest of the codebase.

Comment (ghost) [reply rice-datanode/include/ClientDatanodeProtocolImpl.h]: let's do using-declarations so an import for each. hopefully that will keep things clearer for everyone going forward

Comment (ghost) [reply rice-datanode/include/ClientDatanodeProtocolImpl.h]: @anant12

**Comment by LINKIWI** [comment]
@eddiedugan I've changed all direct references to `using hadoop::hdfs::<imported name>`. Can you make a second pass through?

File: rice-datanode/source/ClientDatanodeProtocolImpl.cc [hunk 4]
```diff
@@ -142,21 +153,33 @@ void ClientDatanodeTranslator::InitServer() {
...
+  // The reason for these binds is because it wants static functions, but we
+  // want to give it member functions
 
   server.register_handler("getReplicaVisibleLength",
-                          std::bind(&ClientDatanodeTranslator::getReplicaVisibleLength, this, _1));
-  server.register_handler("refreshNamenodes", std::bind(&ClientDatanodeTranslator::refreshNamenodes, this, _1));
-  server.register_handler("deleteBlockPool", std::bind(&ClientDatanodeTranslator::deleteBlockPool, this, _1));
+                          std::bind(
+                            &ClientDatanodeTranslator::getReplicaVisibleLength,
+                            this, std::placeholders::_1));
```
Comment (ghost) [inline rice-datanode/source/ClientDatanodeProtocolImpl.cc]: sticking to our convention this should be 
`using std::placeholders::_1;` above

**Review by ghost — COMMENTED** [review]
also not sure about removing commented code but it's up to you

File: rice-datanode/source/ClientDatanodeProtocolImpl.cc (hunk 4 above)
Comment (LINKIWI) [reply rice-datanode/source/ClientDatanodeProtocolImpl.cc]: good catch

**Review by ghost — APPROVED** [review]
approve since it's just one comment ^^

**Pull request merged**